test/CMakeLists.txt

```diff
@@ -23,28 +23,31 @@
  #
  #######################################################################################################################
 
-find_package(Python3 ${LLVM_MINIMUM_PYTHON_VERSION} REQUIRED
-  COMPONENTS Interpreter)
+# Only build if llpc is built as part of xgl
+if(ICD_LLPC_CLIENT_MAJOR_VERSION)
```

Comment:
I guess this means that if LLPC is build as part of anything then it is part of XGL.  It might be better to make the check more specific to XGL being available.  Something like:

```
if (TARGET xgl)
```

should work.

Reply:
While I would very much like to write this, it doesn’t work because the `xgl` target gets defined after the llpc CMakeLists.txt is included, so the `xgl` target is never defined at this point.
I changed to use a `LLPC_IS_STANDALONE` variable if the top-level CMakeLists.txt detects a standalone build.